drizzlepac/haputils/get_git_rev_info.py

```diff
@@ -62,9 +62,17 @@ def print_rev_id(local_repo_path):
             log.info("{}".format(streamline.strip()))
 
         log.info("\n===== Most Recent Commit =====")
-        instream = os.popen('git log |head -1')
-        for streamline in instream:
-            log.info("{}\n".format(streamline.strip()))
+        if 'win' in sys.platform:
+            # for Windows, 'head' command doesn't exist
+            instream = os.popen('git log --oneline |more')
+            for line in instream:
+                log.info("{}\n".format(instream[0].strip()))
```

Comment:
Sorry to be dense, but why is this "for" loop necessary as "line" is never used/

Reply:
Good catch.  Fixed in 9fb037a.